network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/LineImpl.java

```diff
@@ -40,7 +40,7 @@ public Line setR(double r) {
         double oldValue = getResource().getAttributes().getR();
         if (r != oldValue) {
             updateResource(res -> res.getAttributes().setR(r));
-            index.notifyUpdate(this, "r", oldValue, r);
+            index.notifyUpdate(this, "r", null, oldValue, r);
```

Comment:
index.getNetwork().getVariantManager().getWorkingVariantId() instead of null ?

Reply:
We should keep null here, otherwise the following tck will fail : AbstractLineTest.testChangesNotification ...

Reply:
well yes but it is not normal to have a different behavior between R and Xor G1 or G2

Reply:
null is now passed for variantId also for X, B1, B2, G1, G2 (as in powsybl-core ...)